example/querybuilder.go

```diff
@@ -0,0 +1,9 @@
+//go:generate crafting-table qb --dialect=mysql $GOFILE
```

Comment:
In the new version, we remove the example package. Please add a new readme for your command to root project and write your example on it.

Reply:
You mean in `README.md` file right ?

Reply:
Yeah, but you can add a new readme file for your command to the root path of the project. We will make a better document for Crafting-Table soon.

Reply:
I added a section to current readme and I think it's enough for now since we don't have much feature in this version for querybuilder, when we have a new documentation system I will seperate it